app/models/concerns/hyrax/solr_document_behavior.rb

```diff
@@ -11,7 +11,8 @@ module SolrDocumentBehavior
 
     # Add a schema.org itemtype
     def itemtype
-      ResourceTypesService.microdata_type(resource_type.first)
+      types = resource_type || []
+      ResourceTypesService.microdata_type(types.first)
```

Comment:
Incoming. I missed that these methods are tested in the `SolrDocument` context. I was looking for explicit tests of the Behavior mixin.